test/home.js

```diff
@@ -1,20 +1,20 @@
-import 'babel-polyfill';
-import chai from 'chai';
-import chaiHttp from 'chai-http';
-import host from '../index';
+// import 'babel-polyfill';
```

Comment:
Could you remove commented out code and refrain from pushing anything that does not do anything?